docs/configure-custom-values-envoy.md

```diff
@@ -86,42 +80,21 @@ You can configure them using the same syntax as the affinity of Kubernetes. So,
 ```yaml
 envoy:
   affinity:
-    nodeAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        nodeSelectorTerms:
-          - matchExpressions:
-              - key: scalar-labs.com/dedicated-node
-                operator: In
-                values:
-                  - scalardb
```

Comment:
This configuration forces the old too strict recommendation.
We decided to remove this recommendation.